tests/integration/new_relations/test_new_relations.py

```diff
@@ -583,29 +583,40 @@ async def test_invalid_extra_user_roles(ops_test: OpsTest):
         )
 
 
-@pytest.mark.group(1)
+@pytest.mark.group(2)
 @markers.amd64_only  # nextcloud charm not available for arm64
 async def test_nextcloud_db_blocked(ops_test: OpsTest, charm: str) -> None:
```

Comment:
Since a new update to nextcloud charm, its deploy procedure became longer + the fast-forwarding of update status hook was causing the charm to never enter blocked status. To avoid making the charm too long, this test  was moved to a different group.